db/migrate/20150612135546_create_host_status.rb

```diff
@@ -0,0 +1,32 @@
+class CreateHostStatus < ActiveRecord::Migration
+  def self.up
+    create_table :host_status do |t|
+      t.string :type
+      t.integer :status, :default => 0, :null => false
+      t.references :host, :null => false
```

Comment:
I think this should be indexed, as this schema's going to cause a lot of loading of host_status records by host_id.